ngx-fudis/projects/ngx-fudis/src/lib/components/form/select/common/autocomplete/autocomplete.directive.ts

```diff
@@ -79,7 +79,7 @@ export class SelectAutocompleteBaseDirective {
   /**
    * Currently typed input text
    */
-  private _inputText: string | null = null;
+  private _inputText: string = '';
```

Comment:
I _assume_ this was just a side hustle improvement and has little to do with the Esc handling.